PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/serverexplore/ui/AddNewClusterForm.form

```diff
@@ -225,10 +225,10 @@
                     </constraints>
                     <properties>
                       <editable value="false"/>
-                      <enabled value="false"/>
+                      <enabled value="true"/>
                       <model>
                         <item value="Basic Authentication"/>
-                        <item value="No Authentication"/>
+                        <item value="AAD Authentication"/>
```

Comment:
`Azure Active Directory (AAD) Authentication` or `Azure Account`, I prefer the last one.